Marlin/src/gcode/feature/pause/M600.cpp

```diff
@@ -43,73 +48,169 @@
  *  L[distance] - Extrude distance for insertion (positive value) (manual reload)
  *  B[count]    - Number of times to beep, -1 for indefinite (if equipped with a buzzer)
  *
+ * Spool_Swap : Tool change after filament runout
+ *
+ *  T[int]: 0=disable : 1/2/3 Enable the max number of swapping
+ *
+ *  Start Priming : Goto pause, extrude and return 
+ *                : Only one instruction to prime nozzle (one by one only)
+ *  P[bool] : 0 = disable  : 1 = activated
```

Comment:
Priming must be deleted , because easy to integrate for one nozzle but not for multi nozzle 
Please delete it